src/oncall/ui/static/js/oncall.js

```diff
@@ -558,6 +594,31 @@ var oncall = {
                 }
               }
             }
+          },
+          {
+            name: 'users',
+            hint: true,
+            async: true,
+            highlight: true,
+            limit: typeaheadLimit,
+            displayKey: 'name',
+            source: users,
+            templates: {
+              header: function(){
+                return '<h4> Users </h4>';
+              },
+              suggestion: function(resp){
+                return '<div><a href="/dashboard/' + resp.name + '" data-navigo>' + resp.name + '</a></div>';
+              },
+              footer: function(resp){
+                if (usersCt > typeaheadLimit) {
+                  return '<div class="tt-see-all"><a href="/query/' + resp.query + '/users" data-navigo> See all ' + usersCt + ' results for users »</a></div>';
+                }
+              },
+              empty: function(resp){
```

Comment:
Change the styles when "Users" is empty. Do something similar to "Services", where the field is hidden if nothing exists. This should also fix the issue where duplicate "No data found" messages appear on an empty search result (no teams, services, or users).